### ISSUE TITLE
Match level names with multiple words

### ISSUE THREAD

**Issue by ntncmch** [epicentre-msf/hmatch#10]
Add possibility to allow for word-wise comparison when a level name is composed of several words. 

test: Nairobi-City 
ref: Nairobi

At the moment match only with fuzzy matching and dist.max > 3 but should easily match with "wordwise" option.

Certainly need to add a minimal word size argument to avoid multiple false positive.

**Comment by patrickbarks** [comment]
I think we'd also want an option for the user to pass a list of word-tokens not to match because they occur too frequently. E.g. in the COD shapefile there are many repeats of tokens like 'Quartier', 'Camp', 'Village', etc.

But also repeats within multi-term locations that might be harder to notice, e.g. the term "Mangai" in
```
COD | Kwilu | Ipamu | Mangai Etat
COD | Kwilu | Ipamu | Mangai Kimbanguiste
COD | Kwilu | Ipamu | Mangai Paroisse
COD | Kwilu | Ipamu | Mangai Ste Famille
```
We could have a separate utility fn that helps find the most commonly repeated terms, or have an argument specifying the maximum word frequency above which to omit from matching. Either way I think this one will require more 'user oversight'.

**Comment by patrickbarks** [comment]
closed by new function `hmatch_tokens` in 28c0023
